samples/saga/batching/Core_8/WorkProcessor/WorkProcessingHandler.cs

```diff
@@ -0,0 +1,16 @@
+namespace NServiceBus.WorkProcessor;
+
+public class WorkProcessingHandler : IHandleMessages<ProcessWorkOrder>
+{
+    public Task Handle(ProcessWorkOrder message, IMessageHandlerContext context)
+    {
+        Console.WriteLine($"Processing work order '{message.WorkOrder}'");
+        Thread.Sleep(Random.Shared.Next(500, 1000));
+
+        return context.Send(new WorkOrderCompleted
```

Comment:
Use Reply instead of Send, and use try..catch for non-transient errors like work order no longer exists or some functional error. See other comment.

Reply:
Reply makes sense, but not sure about Try...Catch. What would you catch in a handler context that makes sense for a sample?

Reply:
Logical errors raised via exceptions would require a try..catch as those would be expected return/response states.

Alternatively, maybe just do 2 different replies not based on a catch but just an imaginary validation. Done in the same message but a different state?

Reply:
Used a status to report simulated errors back to the saga.